core/services/offchainreporting/example-job-spec.json

```diff
@@ -0,0 +1,36 @@
+{
+    "observationTimeout": "10s",
+    "contractAddress": "0x613a38AC1659769640aaE063C651F48E0250454C",
```

Comment:
Needs a "bootstrap mode" which is potentially even a completely different type of job